.github/workflows/docker-backend.yml

```diff
@@ -9,19 +9,13 @@ on:
       # Name of service. Empty for root service, needs to be unique for services from subfolders. 
       # Will be used as a last part of container path and matched with version tags
       name:
-        required: false
+        required: true
         type: string
-        default: ""
 
       path:
         required: false
```

Comment:
Due to the step that validates the occurrence of inputs, this is required, too, while it should not. This needs more adjustments I think ^^

Reply:
It's more that we don't need that check now :)